main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"os"
+
+	"github.com/yelyzaveta-mykhalik/brainfuck/brainfuck"
+)
+
+func main() {
+
+	code := "++++++++++[>+++++++>++++++++++>+++>+<<<<-]>++.>+.+++++++..+++.>++.<<+++++++++++++++.>.+++.------.--------.>+.>."
+
+	compiler := brainfuck.NewCompiler(string(code))
```

Comment:
`code` is already a string.

Reply:
I no longer use this part of code and read brainfuck program from file